packages/ui/src/kit/VAccordion/VAccordion.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div :class="accordionClassName">
+    <v-accordion-item
+      v-for="(item, index) in items"
+      :key="item.id"
+
+      :class="classNames.item"
+    >
+      <template v-slot:item>
+        <slot name="item" v-bind:item="item"></slot>
```

Comment:
not sure, what does mean `item`.
and as i understand, accordion - it's just a title with collapse/expand content.
why just make it simpler? 
with creating accordion container only with styles and child container with :title prop and collapse/expand logic of passed <slot/>

Reply:
1. Recent instructions require several active items. And we can user [tab approach](https://github.com/endpass/endpass-core/tree/master/packages/ui/src/kit/VTab)
2. The title should be a slot, because we have a complex header for the item in the design  with various components (title, tag, status)

Reply:
@dudiq `item` is an `item` of accordion `items`. I choose generic name because it have sense in scope of `accordion`. In other words, `item` in `accordion` = `accordionItem`. It's correct.

> why just make it simpler?

External component should be flexible at it's first. Only then - simplicity, and I must say, slots are pretty simple. I might want to define my own custom item, and many developers of poor-quality components just don't provide any opportunity to do it. It's bad.

> with creating accordion container only with styles and child container with :title prop and collapse/expand logic of passed 

Inflexible. It can take more code when reuse.

Reply:
@kazurus yes, I planned to make title a slot.

Reply:
@ifiri please, look [this table approach](https://github.com/endpass/endpass-core/tree/master/packages/ui/src/kit/VTable) and create accordion in same way

Reply:
@kazurus why my approach is bad? It's pretty good.

Reply:
@kazurus you suggest me drop scoped slots? In this case, slots are senseless, because we will not be able to create custom layout of item without access to it's scope. We need `v-bind` in slots. Then, you suggest drop multiple slots in VAccordion component in favor of default one. This means, we will forced to use all parts of component. It forced us to use extra code, like in provided `VTable` example. Is this at least one objective reason for that? Compare:
```
<v-accordion
  :items="addresses"
>
  <template v-slot:item="{ item }">
    <profile-address></profile-address>
  </template>

  <template v-slot:content>
    Lorem ipsum dolor sit amet, consectetur adipisicing elit
  </template>
</v-accordion>
```
In example below, we have one default slot. So it means, we should explicitly provide content of entire accordeon, just like in VTable component.
```
<v-accordion>
  <v-accordion-item
    foo="bar"
  >
    <v-accordion-title>
      <profile-address></profile-address>
    </v-accordion-title>

    <v-accordion-content>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit
    </v-accordion-content>
  </v-accordion-item>
  
  <v-accordion-item
    foo="bar"
  >
    <v-accordion-title>
      <profile-address></profile-address>
    </v-accordion-title>

    <v-accordion-content>
      Lorem ipsum dolor sit amet, consectetur adipisicing elit
    </v-accordion-content>
  </v-accordion-item>
</v-accordion>
```
I understand reasons of architecture like that, it's pretty good when we deal with tables. Tables are flexible by design, so we should be able to set cells and rows as we want. But accordion is not the same. Accordion have robust, simple structure and there are no reasons to force final user write extra code every time. It's excessively.

Is my example worse than example with default slots? This is not a claim, I just want to understand why you prefer second approach :)